content/Video_Display.md

```diff
@@ -179,15 +179,15 @@ instead.
 
 ### FF48 - OBP0 (Object Palette 0 Data) (R/W) - Non CGB Mode Only
 
-This register assigns gray shades for sprite palette 0. It works exactly
-as BGP (FF47), except that the lower two bits aren't used because
-sprite data 00 is transparent.
+This register assigns gray shades to the color indexes of the OBJs that use this palette. It works exactly
+like BGP (FF47), except that the lower two bits aren't used because
```

Comment:
Maybe "aren't used" could be changed to "are ignored"?

Reply:
Yep. Fixed.

Reply:
Likewise line 189 for consistency.

Reply:
Fixed